tensorflow/compiler/xla/service/gpu/ir_emitter.cc

```diff
@@ -55,14 +56,17 @@ using llvm_ir::SetToFirstInsertPoint;
 namespace gpu {
 
 IrEmitter::IrEmitter(const HloModuleConfig& hlo_module_config,
-                     IrEmitterContext* ir_emitter_context, bool is_nested)
+                     IrEmitterContext* ir_emitter_context, bool is_nested,
+                     llvm_ir::LLVMTargetFeatures* llvm_target_features)
     : ir_emitter_context_(ir_emitter_context),
       module_(ir_emitter_context->llvm_module()),
       b_(module_->getContext()),
       bindings_(ir_emitter_context->hlo_module(),
                 &ir_emitter_context->buffer_assignment(), &b_, module_,
                 is_nested),
-      hlo_module_config_(hlo_module_config) {
+      hlo_module_config_(hlo_module_config),
+      llvm_target_features_(*llvm_target_features) {
```

Comment:
In case `IrEmitterUnnested` is constructed without an `llvm_target_features`, `llvm_target_features_` field in `IrEmitter` would become a garbage reference.